src/OSPSuite.UI/Controls/UxHistogramControl.cs

```diff
@@ -79,11 +79,11 @@ public void Plot(ContinuousDistributionData distributionData, DistributionSettin
          if (!distributionData.IsRangeSpecified)
             return;
 
+         var templateView = SeriesTemplate.View as BarSeriesView;
```

Comment:
This is the change. The other files are introducing the new Histogram Starter and refactoring starters so that they do not receive their presenter on construction, but only when the started.